src/Components/Detail/Detail.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './Detail.scss';
+import '../../Styles/common.scss';
+import '../../Styles/reset.scss';
+
+class Detail extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      product: {},
+    };
+  }
+  getInfo = () => {
+    fetch('/data/mockdata.json', {
+      method: 'GET',
+    })
+      .then(products => products.json())
+      .then(products => {
+        this.setState({
+          product: products.filter(product => product.id === 2),
+        });
+      });
+  };
+  componentDidMount() {
+    this.getInfo();
+  }
+  render() {
+    const { product } = this.state;
+    console.log(product);
+    return (
+      <div className="Detail">
```

Comment:
```suggestion
      <div className="detail">
```
camelCase에 따라 최상단 컴포넌트 또한 위와 같이 사용해주시면 좋겠습니다!

Reply:
넵 수정했습니다!